src/api/posts.ts

```diff
@@ -0,0 +1,60 @@
+import { BASE_URL } from './api';
+
+export async function getUserPosts(userId: number): Promise<Post[]> {
+  const response = await fetch(`${BASE_URL}/posts${userId === 0 ? '' : `?userId=${userId}`}`);
+  const res = await response.json();
+
+  if (res.Response === 'False') {
+    throw new Error('Something went wrong.');
+  } else {
+    return res;
+  }
```

Comment:
else is redundant here
```suggestion
  }
    
  return res;
```